### PR TITLE
Fiixed connection cr length by replacing instance id with current timestamp

### PR DESCRIPTION
**Pull request by redhatHameed**
issue :  
Screen Shot 2021-09-23 at 4 22 20 PM
https://issues.redhat.com/browse/DBAAS-159

### CODE REVIEW

**Review by yzhao583 — CHANGES_REQUESTED** [review]
@redhatHameed Looks like you are using older version code, could you fetch upstream?

**Comment by redhatHameed** [comment]
@yzhao583 I rebase can you look now

File: src/components/instanceTable.jsx [hunk 1]
```diff
@@ -110,7 +110,7 @@ class InstanceTable extends React.Component {
       kind: 'DBaaSConnection',
       metadata: {
         //k8s only accept lowercase metadata.name and add instanceID to avoid same name
-        name: `${this.state.selectedInstance.name.toLowerCase()}-${this.state.selectedInstance.instanceID.toLowerCase()}`,
+        name: `${this.state.selectedInstance.name.toLowerCase()}-${Date.now()}`,
```
Comment (yzhao583) [inline src/components/instanceTable.jsx:113]: In line 79, can you change `rowList.push({ cells: [rowData.instanceID, `${rowData.name}-${rowData.instanceID}`] })` to `rowList.push({ cells: [rowData.instanceID, `${rowData.name}-${Date.now(}`] })`

Comment (redhatHameed) [reply src/components/instanceTable.jsx:113]: I think in tabel we can show combination of name-instanceID because only for display purpose WDYT ?

Comment (yzhao583) [reply src/components/instanceTable.jsx:113]: @redhatHameed If the names are different, would it confuse user?

Comment (redhatHameed) [reply src/components/instanceTable.jsx:113]: not much aware about complete code where we are displaying the names different? 

I see 
Screen Shot 2021-09-23 at 5 18 04 PM
I would also suggest to remove the instanceid from instance name WDYT ? cc: @jeremyary

Comment (redhatHameed) [reply src/components/instanceTable.jsx:113]: nvm, I understand what you mean. it will display once we connect.

Comment (redhatHameed) [reply src/components/instanceTable.jsx:113]: thanks updated at line 79

File: src/components/instanceTable.jsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ class InstanceTable extends React.Component {
     let rowList = []
     if (data && data.length > 0) {
       _.forEach(data, (rowData) => {
-        rowList.push({ cells: [rowData.instanceID, `${rowData.name}-${rowData.instanceID}`] })
+        rowList.push({ cells: [rowData.instanceID, `${rowData.name}`] })
```
Comment (yzhao583) [inline src/components/instanceTable.jsx:79]: @redhatHameed We don't need ``${rowData.name}`` here, it should just be `rowData.name`

**Review by yzhao583 — APPROVED** [review]
lgtm!

**Pull request merged**